src/lib_ccaligner/read_wav_file.cpp

```diff
@@ -344,14 +342,12 @@ int WaveFileData::processStreamHeader()
             }
         }
     }
-    LOG("Error occurred while processing stream header!");
-    FATAL(EXIT_UNKNOWN, "Error occurred while processing stream header!");
-    return -1;  //some error; more robust exit errors coming soon
+    FATAL(UnknownError) << "Error occurred while processing stream header!";
```

Comment:
```
/Users/travis/build/saurabhshri/CCAligner/src/lib_ccaligner/read_wav_file.cpp:346:1: warning: control may reach end of non-void function [-Wreturn-type]
}
```
^
/Users/travis/build/saurabhshri/CCAligner/src/lib_ccaligner/read_wav_file.cpp:498:1: warning: control may reach end of non-void function [-Wreturn-type]
}

Reply:
FATAL will abort the program (there's not catch block) it won't reach the end of the function. I believe it's a false positive.